pkg/sdk/logging/api/v1beta1/syslogng_types.go

```diff
@@ -66,7 +66,10 @@ type GlobalOptions struct {
 	// deprecated use stats/freq from 4.1+
 	StatsFreq *int `json:"stats_freq,omitempty"`
 	// TODO switch to this by default
```

Comment:
Is this TODO item still valid?

Reply:
no and it's not a good idea to write todos into places that we use to generate docs from (note for myself)
fixed in 67a352c3